src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx

```diff
@@ -0,0 +1,33 @@
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+
+import { Spacing } from '../../../styles';
+
+export default function SubtestsBreadcrumbs() {
+  const breadcrumbs = [
+    <div key='1'></div>,
+    <Link key='2' href='/'>
+      Home
+    </Link>,
+    <Link key='3' href='/'>
```

Comment:
@julienw Here I was thinking of having a function which generates the previous URL. Do you have other suggestions for me?